Makefile

```diff
@@ -18,3 +22,20 @@ run-docker:
         -e VIA_URL=http://localhost:9080 \
 		-p 9081:9081 \
 		hypothesis/proxy-server:$(DOCKER_TAG)
+.PHONY: dev
+dev:
+	docker-compose run --service-ports proxy-server
+
+.PHONY: test
+test:
+	docker-compose run --service-ports proxy-server-test
+
+.PHONY: ci-test
```

Comment:
This may or may not stay depending on how we call the tests in the Jenkin's file.